airbyte-webapp/src/components/connection/CatalogTree/next/StreamDetailsPanel/StreamFieldsTable/StreamFieldsTable.module.scss

```diff
@@ -59,35 +60,45 @@ $cell-left-padding: variables.$spacing-xl + variables.$spacing-sm;
   padding: 0;
 
   &:first-child {
-    padding-left: $cell-left-padding;
+    padding-left: $cell-padding;
     border-radius: 0;
+    min-width: 130px;
+  }
+
+  &:last-child {
+    padding-right: $cell-padding;
   }
 }
 
 .textCell {
   @extend %bodyCell;
 
   color: colors.$dark-blue;
+  max-width: 0;
 }
 
 .dataTypeCell {
   @extend %bodyCell;
 
+  width: $cell-fixed-width;
```

Comment:
The data type cell should maybe have a min-width and max-width. The max width should be up to 200px or so to fit the new data types introduced in #21757 (notable `Timestamp With Timezone`

Reply:
Double-checked: if `100px` is not enough - the cell will grow to fit the content
<img width="861" alt="image" src="https://user-images.githubusercontent.com/20929344/215474634-21dc53ae-4ec4-4f63-ba51-efa58969be65.png">